data-raw/render_available_datasets.R

```diff
@@ -0,0 +1,21 @@
+# render available datasets table
+
+envi <- ls(getNamespace("covidregionaldata"), all.names = TRUE)
```

Comment:
shouldn't this just use the package function?

Reply:
I just copied this over from the previous version...

Reply:
sorry as in call `get_avaiable_data(render = TRUE)`?